Tests/DataParser/test_dataParser.py

```diff
@@ -0,0 +1,116 @@
+import os
+import json
+import xmltodict
+import zipfile
+import pandas as pd
+from pathlib import Path
+import pytest
+import sys
+sys.path.append("../../DataParser")
+import filecmp
+import dataParser
+
+testFiles = "./testFiles"
+truth_folder = "./testFiles/truth_data"
+
+__TESTS__ = 4 
+
+def build_testdata():
+	"""
+	This method builds the gold directory structure for the files added
+	in the test folder of the data path. This created directory structure 
+	will be used while comparing the test.
+	"""
+	os.system("rm -rf ./testFiles/truth_data/*")
+	for i in range(1,__TESTS__+1):
+		print("Preparing Test :",i)
+		os.system("rm -rf ./testFiles/test"+str(i)+"/generated")
```

Comment:
As said in previous comment, please remove all these rm -rf commands, these are not safe.